poulpe-model/src/test/java/org/jtalks/poulpe/model/entity/PoulpeUserTest.java

```diff
@@ -31,4 +33,12 @@ public Group groupWithId(long id) {
         return group;
     }
 
+    @Test
+    public void theEntityFieldsShouldBeSerialized(){
+        PoulpeUser user = PoulpeUser.withId(1);
```

Comment:
In the PoulpeUser we need all the fields filled. Otherwise when we check before and after we'll get the same results (null) even if serialization doesn't work.